libraries/rush-lib/scripts/copyEmptyModules.js

```diff
@@ -0,0 +1,63 @@
+'ues strict';
+
+const { FileSystem } = require('@rushstack/node-core-library');
+
+const JS_FILE_EXTENSION = '.js';
+const DTS_FILE_EXTENSION = '.d.ts';
+
+module.exports = {
+  runAsync: async ({ scopedLogger: { terminal }, heftConfiguration: { buildFolder } }) => {
+    // This script looks through the `lib-esnext` folder for modules that only
+    // contain types, and copies them to the `lib` folder
```

Comment:
Could you make this comment more detailed? Looking at the file, I really have no idea what it's doing. (Also, some basic Jest tests might be enlightening, if it's not too difficult to set up.)

Reply:
Could you make this comment more detailed? Looking at the file, I really have no idea what it's doing. (Also, some basic Jest tests might be enlightening, if it's not too difficult to set up.)

Reply:
Would it make sense for copyEmptyModules.js to be provided by the `webpack-deepimports-com-plugin`?